openquake/calculators/event_based_risk.py

```diff
@@ -179,6 +173,12 @@ def zeroN():
                         monitor.ela_dt))
 
             # agg losses
+            glosses = out.event_loss_per_asset.sum(axis=1)
+            ilosses = out.insured_loss_per_asset.sum(axis=1)
+            agglosses = {
+                rupid: numpy.array([gloss, iloss], F32)
+                for rupid, gloss, iloss in zip(out.rupids, glosses, ilosses)
```

Comment:
You have a point here, but since I have another branch were all the names have changed again, I will wait for the other pull request.